_config.yml

```
@@ -50,3 +52,26 @@ defaults:
         link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
       - title: Spécifications de l'architecture de l'information
         link: https://conception.canada.ca/architecture/specifications-contenu-architecture-information-canada.html
+  - scope:
+      path: "configurations-conception-communes/*.md"
+    values:
+      breadcrumbs:
+      - title: À propos de Canada.ca
+        link: https://www.canada.ca/fr/gouvernement/a-propos.html
+      - title: Système de conception de Canada.ca
+        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
+      - title: Bibliothèque de modèles et de configurations de conception
+        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/bibliotheque-modeles.html
+  - scope:
+      path: "configurations-conception-communes/pied-page-general/*.md"
+    values:
+      breadcrumbs:
+      - title: À propos de Canada.ca
+        link: https://www.canada.ca/fr/gouvernement/a-propos.html
+      - title: Système de conception de Canada.ca
+        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
+      - title: Bibliothèque de modèles et de configurations de conception
+        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/bibliotheque-modeles.html
+      - title: Pied de page général
+        link: https://conception.canada.ca/configurations-conception-communes/pied-page.html
+      permalink: /configurations-conception-communes/:basename:output_ext
```

```suggestion
  - scope:
      path: "configurations-conception-communes/pied-page-general/*.md"
    values:
      breadcrumbs:
      - title: À propos de Canada.ca
        link: https://www.canada.ca/fr/gouvernement/a-propos.html
      - title: Système de conception de Canada.ca
        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
      - title: Bibliothèque de modèles et de configurations de conception
        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/bibliotheque-modeles.html
      - title: Pied de page général
        link: https://conception.canada.ca/configurations-conception-communes/pied-page.html
      permalink: /configurations-conception-communes/:basename:output_ext
```
Let's keep the change to the config files to the bare minimum. The master branch will already have the previous mods by the time we merge this one.

---

Okay